pwa/app/routes/api-docs.tsx

```diff
@@ -0,0 +1,304 @@
+            <Link to="/api-docs/webhooks">Webhooks</Link>
+          </h3>
+          <p>
+            The TBA API also includes support for{' '}
+            <a href="https://en.wikipedia.org/wiki/Webhook">Webhooks</a>, or
+            HTTP callbacks. When an item of interest changes, TBA can send a
+            HTTP request to your server, allowing your application to react
+            instantly to the change. This can save both your client and our
+            server time and processing power, as it can reduce the need to poll
+            the API. See{' '}
+            <Link to="/api-docs/webhooks">our webhook documentation page</Link>{' '}
+            for more information.
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">
+            <Link to="/api-docs/apiwrite">Write API (v1)</Link>
+          </h3>
+          <p>
+            The Blue Alliance provides a Write API, called the Trusted API,
+            which allows users to import data to The Blue Alliance for archival.
+            For example, many offseason events use the Trusted API to provide
+            real-time match results to their audience. To get started, you need
+            to <Link to="/request/apiwrite">request access tokens</Link> for
+            your desired event. These need to be used when constructing each
+            request.
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">
+            <a href="https://github.com/the-blue-alliance/the-blue-alliance-data">
+              Data Archives
+            </a>
+          </h3>
+          <p>
+            If you&apos;re looking to run SQL queries over the TBA database, you
+            can use this{' '}
+            <a href="https://www.thebluealliance.com/bigquery">BigQuery</a>{' '}
+            dataset. This contains a full replica of the TBA database, so the
+            possibilities are endless!
+          </p>
+          <p>
+            If these APIs are too complex for your application, The Blue
+            Alliance also provides periodic data archives in comma separated
+            value (CSV) format. The data provided here is less detailed, but is
+            simpler to use in someething like a spreadsheet.
+          </p>
+          <p>
+            You can find these data archives on
+            <a href="https://github.com/the-blue-alliance/the-blue-alliance-data">
+              {' '}
+              The Blue Alliance&apos;s TBA Data Repository
+            </a>
+          </p>
+        </div>
+
+        <div>
+          <h3 className="text-2xl">Developer Guidlines</h3>
+          <p>
+            We’re excited to see you building things on top of The Blue
+            Alliance’s APIs! We love seeing the creativity and utility in these
+            projects. We have a few developer guidelines to help guide your
+            work.
+          </p>
+
+          <div>
+            <h4 className="text-xl">1. Brading</h4>
```

Comment:
Did you not copy-paste the page? Why is this spelled wrong?

Reply:
💀